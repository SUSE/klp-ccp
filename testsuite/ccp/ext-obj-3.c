static int ee_o = 0;

void pu_f(void)
{
	ee_o;
}
