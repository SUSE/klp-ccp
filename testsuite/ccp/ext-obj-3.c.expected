extern int ee_o;

void klpp_pu_f(void)
{
	ee_o;
}
